qcschema/dev/wavefunction/result_wavefunction.py

```diff
@@ -0,0 +1,78 @@
+"""
+The primary specified return  wavefunction quantities.
+"""
+
+result_wavefunction = {}
+
+# Orbitals
+result_wavefunction["return_result_orbitals_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin orbitals in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_orbitals_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin orbitals in the AO basis",
+    "items": {"type": "number"}
+}
+
+# Density
+result_wavefunction["return_result_density_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin density in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_density_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin density in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+# Fock matrix
+result_wavefunction["return_result_fock_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin Fock matrix in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_fock_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin Fock matrix in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+# Eigenvalues
+result_wavefunction["return_result_eigenvalues_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin orbital eigenvalues",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_eigenvalues_b"] = {
+    "type": "array",
+    "description": "SCF beta-spin orbital eigenvalues",
+    "items": {"type": "number"}
+}
+
+
+# Occupations
+result_wavefunction["return_result_occupations_a"] = {
+    "type": "array",
+    "description": "SCF alpha-spin orbital occupations",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_occupations_b"] = {
+    "type": "array",
+    "description": "SCF beta-spin orbital occupations",
```

Comment:
These are not only SCF level occupations?

Reply:
TODO fix this